lib/modules/platform/gitlab/index.ts

```diff
@@ -1340,3 +1347,37 @@ export async function filterUnavailableUsers(
   }
   return filteredUsers;
 }
+
+export async function expandGroupMembers(
+  reviewersOrAssignees: string[]
+): Promise<string[]> {
+  const expandedReviewersOrAssignees: string[] = [];
+  const normalizedReviewersOrAssigneesWithoutEmails: string[] = [];
+
+  // Skip passing user emails to Gitlab API, but include them in the final result
+  for (const reviewerOrAssignee of reviewersOrAssignees) {
+    if (reviewerOrAssignee.indexOf('@') > 0) {
```

Comment:
```suggestion
    if (reviewerOrAssignee.indexOf('@') > 1) {
```
otherwise you filter all which are starting with `@` too

Reply:
> 0 should be correct because if it starts with an @, it would be at index 0. So I want greater than index 0 (incase we have `a@email.com`, which would be index 1.